tests/in/force_point_size_vertex_shader_webgl.param.ron

```diff
@@ -4,7 +4,7 @@
             version: 300,
             is_webgl: true
         ),
-        writer_flags: (bits: 16),
+        writer_flags: ("TEXTURE_SHADOW_LOD | INCLUDE_UNUSED_ITEMS | FORCE_POINT_SIZE"),
```

Comment:
```suggestion
        writer_flags: ("FORCE_POINT_SIZE"),
```

I think it's because of this.

Reply:
Wow, not sure how I came up with this combination.
In any case, this should be `CLAMP_FRAG_DEPTH`, right?
https://github.com/daxpedda/naga/blob/cc985396dadc95039a0f3d5cfc818a6b4d5eba6b/src/back/spv/mod.rs#L604-L620

Reply:
The `WriterFlags` you linked to is for the SPV backend, the one in question here is for the GLSL backend.

Reply:
> The `WriterFlags` you linked to is for the SPV backend, the one in question here is for the GLSL backend.

Right ... thank you!